src/requests/sessions.py

```diff
@@ -600,6 +600,11 @@ def get(self, url, **kwargs):
 
         kwargs.setdefault("allow_redirects", True)
         return self.request("GET", url, **kwargs)
+   
+    def send(self, url, **kwargs):
+
+        kwargs.setdefault("allow_redirects", False)
+        return self.request("SEND", url, **kwargs)
```

Comment:
**Suggestion:** The new 'send' method in sessions.py is not following the HTTP methods convention. Consider renaming it to a proper HTTP method or make it more generic if it's intended to handle multiple methods.
```suggestion
    def custom_request(self, method, url, **kwargs):
        kwargs.setdefault("allow_redirects", False)
        return self.request(method.upper(), url, **kwargs)
```